test/functional/Valhalla/build.xml

```diff
@@ -54,7 +54,7 @@
 		<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" encoding="ISO-8859-1">
 			<src path="${src}"/>
 			<src path="${transformerListener}" />
-			<compilerarg line='--add-exports java.base/jdk.internal.misc=ALL-UNNAMED' />
+			<compilerarg line='--add-exports java.base/jdk.internal.misc=ALL-UNNAMED -XDignore.symbol.file' />
```

Comment:
due to the use of Unsafe, otherwise there will be lots of warning message like this: "warning: Unsafe is internal proprietary API and may be removed in a future release" when you use unsafe.

Reply:
Please remove the `-XDignore.symbol.file` option